website/templates/issue.html

```diff
@@ -295,8 +294,8 @@ <h4>OS Version: {{ os_version }}</h4>
                                 on Github</a>
                             {% else %}
                             {% if request.user.is_superuser or request.user == object.user %}
-                            <a href="{{ object.domain.github }}/issues/new?title={{ object.description|urlencode }}&body=![0]({{ object.screenshot.url }})%20http://bugheist.com/issue/{{ object.id }}"
```

Comment:
same here - keep first image if screenshot and if multiple images show the first